cookies.css

```diff
@@ -0,0 +1,40 @@
+    .cookie {
+        position: fixed;
+        bottom: 0;
+        padding: 30px;
+        width: 100%;
+        min-height: 115px;
+        background-color: var(--color-blue);
+        color: var(--color-grey);
+        text-align: center;
+        border-top: 1px solid var(--color-dark);
+    }
+    
+    .cookie__container {
+        height: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+    }
+    
+    .cookie__container--button {
+        background-color: white;
+        align-items: center;
+        width: 150px;
+        height: 40px;
+        margin-top: 10px;
+        text-decoration: none;
+        outline: none;
+        border-radius: 20px;
+    }
+    
+    @media (min-width: 768px) {
+        .cookie {}
```

Comment:
do usunięcia ;)

Reply:
w sensie ta pusta deklaracja cookie{}

Reply:
Poprawione :) miałem to usunąć a zapomniałem :D